core/src/main/java/hudson/cli/DeleteViewCommand.java

```diff
@@ -52,50 +50,35 @@ public String getShortDescription() {
     }
 
     @Override
-    protected int run() throws Exception {
-
-        boolean errorOccurred = false;
-
-        // Remove duplicates
-        final HashSet<String> hs = new HashSet<>(views);
+    protected void checkExists(View viewObject, String viewName, String viewType) throws Exception {
+        if (viewObject == null) {
+            throw new IllegalArgumentException("View name is empty");
```

Comment:
Maybe this error message was not valid previously? The name might not be empty and yet the view doesn't exist.